Gruntfile.js

```diff
@@ -340,6 +329,10 @@ module.exports = function(grunt) {
 			}
 		},
 		run: {
+			npm_run_imports: {
```

Comment:
I thought about this @stephenmathieson, but this required developer documentation update at various places. 

Our docs are tied to steps like `grunt dev`, `grunt build` and `grunt translate`. So the only way to trigger any current npm script is via `grunt-run`, like above.

You concern of what confusion happened in above #1252 etc., will not re-occur as I have already stripped out multiple references to same task and refactored it.

As you can see from most of the chore PR's, once we more to pure npm scripts, it will be easier to update documentation when we have change all of the grunt tasks. So the answer is NO for the time being.

As if I do this change above now, a developer running `grunt build` will yield different results to `npm run build`.

Reply:
That may be true about `grunt-run` and so on, so I can understand not wanting to take all of that on. As an aside, though, I wouldn't let developer docs stop you from making any changes; we can always update the docs. I'm happy to help with that!

Reply:
Some of my `chore` work in axe-core is moving away from grunt to npm scripts. I will make the switch to above recommendation in a different PR when time is right :).